script/get-dependencies.sh

```diff
@@ -28,14 +28,15 @@ for dependency_path in "${dependencies[@]}"; do
     then
       repo_name="daml-finance"
       package_name=`echo ${package_name//-/.} | sed -e "s/\b\(.\)/\u\1/g"`
-      download_path="https://github.com/DACH-NY/${repo_name}/releases/download/${package_name}/${version}/${file_name}"
+      package_name=`echo ${package_name/Refdata/RefData}`
```

Comment:
unsure if `RefData` here should be hardcoded

Reply:
Needs to, as the automatic conversion from package name to file path ends up with `Refdata` instead of `RefData` leading to a 404. I don't see a better way to do this.